easy_rec/python/model/rank_model.py

```diff
@@ -54,9 +57,24 @@ def _output_to_prediction_impl(self,
     return prediction_dict
 
   def _add_to_prediction_dict(self, output):
-    self._prediction_dict.update(
-        self._output_to_prediction_impl(
-            output, loss_type=self._loss_type, num_class=self._num_class))
+    prediction_dict = self._output_to_prediction_impl(
+        output, loss_type=self._loss_type, num_class=self._num_class)
+    self._prediction_dict.update(prediction_dict)
+    if self._loss_type == LossType.CLASSIFICATION:
+      if 'probs' in prediction_dict:
+        self._forward_rank_predict(prediction_dict['probs'])
+    elif self._loss_type in [LossType.L2_LOSS, LossType.SIGMOID_L2_LOSS]:
+      if 'y' in prediction_dict:
+        self._forward_rank_predict(prediction_dict['y'])
+
+  def _forward_rank_predict(self, tensor):
+    try:
+      op = tf.get_default_graph().get_operation_by_name('rank_predict')
```

Comment:
add some method description, _forward_rank_predict could also be used in subclass, so it is better to named to  forward_rank_predict

Reply:
Comment added. This function is not supposed to be used in subclass but only in `RankMode._add_to_prediction_dict`, which recognizes the prediction node and forward it to RTP.